django_cyverse_auth/models.py

```diff
@@ -224,7 +223,7 @@ def lookupSessionToken(request):
     """
```

Comment:
I removed the circular reference that loads token (definition found on line 67) as authtoken.  Other than removing circular reference, there were no significant changes made to that file.  I would opt to keep the comments the same as the class name hasn't been changed.